### PR TITLE
Add ga tracking to config

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

We want admins to be able to set a Google Analytics tracking code so it's not an ENV var.

### CODE REVIEW

File: app/lib/constants/site_config.rb [hunk 1]
```diff
@@ -113,8 +113,8 @@ module SiteConfig
         description: "The \"Client Secret\" portion of the GitHub Oauth Apps portal",
         placeholder: ""
       },
-      ga_view_id: {
```
Comment (mstruve) [inline app/lib/constants/site_config.rb:116]: Was this just unused?

Comment (benhalpern) [reply app/lib/constants/site_config.rb:116]: Yeah, this was part of the functionality for using the Google Analytics API for tallying total page views, which we no longer use. So this is just dead functionality.

File: app/assets/javascripts/initializers/initializeBaseTracking.js.erb [hunk 2]
```diff
@@ -11,7 +11,7 @@ function initializeBaseTracking() {
     addedGA = true;
     wait++;
     if (window.ga && ga.create) {
-      ga('create', '<%= ApplicationConfig["GA_TRACKING_ID"] %>', 'auto');
+      ga('create', document.body.dataset.gaTracking, 'auto');
```
Comment (benhalpern) [inline app/assets/javascripts/initializers/initializeBaseTracking.js.erb:14]:
```suggestion
      ga('create', '<%= ApplicationConfig["GA_TRACKING_ID"] %>', 'auto');
```

**Comment by benhalpern** [comment]
I modified this to actually _not yet_ use this value in the document, because there _could_ be a scenario where we roll this out and it's not yet available in the view but the functionality is changed in the JS....

So I'm going to do the change in JS as an additional step 24 hours after merging this (aka once all caches would have cleared)

**Pull request merged**